stable/jenkins/templates/jenkins-backup-cronjob.yaml

```diff
@@ -97,6 +97,15 @@ spec:
           {{- end }}
           {{- end }}
           {{- end }}
+{{- if .Values.backup.usePodSecurityContext }}
+          securityContext:
+            runAsUser: {{ default 0 .Values.backup.runAsUser }}
+{{- if and (.Values.backup.runAsUser) (.Values.backup.fsGroup) }}
+{{- if not (eq (int .Values.backup.runAsUser) 0) }}
+            fsGroup: {{ .Values.backup.fsGroup }}
```

Comment:
Do you see a use case where runAsUser for backup and Jenkins would be different? Otherwise we could re-use the existing value `master.runAdUser`. Then we only need to configure the user in one place.

Reply:
Unfortunately, i have absolutely no idea atm. I have never needed that so cant tell if it is crucial to have them separated. For my case now, there shouldnt be any difference, but didnt check what will happen if i set the runAsUser and fsGroup in master values, untill few days ago i was working with older version ;) If there is no big problem i would leave 2 separate options, but probably either way works for me. Dunno about others.

Reply:
I suggest to use master.runAsUser

Could you change it and merge changes from Master into your branch?